### PR TITLE
Improve language, add tippy

### PR DESCRIPTION
**Pull request by lkubb**
I ran everything through an LLM and synthesized its output with what I preferred, this is the result. Imho, it improves on conciseness and clearness quite a bit.

Additionally adds `sphinx-tippy` for sneaky peeking.

After this, I'm fine with pulling the trigger.

Still need to adjust the default README somehow though.

### CODE REVIEW

File: docs/ref/concepts.md [hunk 1]
```diff
@@ -1,74 +1,67 @@
...
-* After the Broadcom event, the push really came to shove, and it was said that there [won't be any deprecation period](https://www.youtube.com/watch?v=CubGR8rTy3Y&t=245s).
-* In January 2024, the [list of modules to be extracted](https://github.com/saltstack/great-module-migration) was opened for comments. Take your time to read it.
-* The modules are being migrated into the [salt-extensions](https://github.com/salt-extensions) community org.
-* In February 2024, the [great module purge PR](https://github.com/saltstack/salt/pull/65971) was created.
-* In April 2024, the great module purge PR was merged, making 3008 the target release.
+The SaltStack team has embarked on a significant transformation known as the "Great Module Migration." This initiative involves moving many modules, which were previously part of the core Salt distribution, into separate extensions. While this shift promises to streamline Salt’s core, it also means that many modules will no longer be maintained directly by the Salt team. Instead, their future now depends on community contributors.
```
Comment (max-arnold) [inline docs/ref/concepts.md]: s/SaltStack/Salt Project/ (if I remember correctly, the SaltStack brand belongs to VMware/Broadcom after the acquisition)

File: docs/ref/layout.md [hunk 2]
```diff
@@ -50,46 +50,46 @@ Only present when {question}`workflows` == `enhanced`.
...
 :::
 ### `docs/ref`
-Directory containing autogenerated documentation of your modules. Usually, you don't need to touch this directory, but it could be used e.g. for a configuration reference document.
+Directory containing autogenerated module documentation. Typically does not require manual updates, but can be used for custom documents, like a configuration reference.
 
 :::{path} docs/topics
 :::
 ### `docs/topics`
-Intended to contain high-level guides regarding your Salt extension, such as `Configuration`.
+Intended to hold high-level guides related to your Salt extension, such as `Configuration`. By default, includes a guide on `Installation`.
```
Comment (max-arnold) [inline docs/ref/layout.md]: By default, includes a guide on `Installation` -> By default, includes an `Installation` guide

IMO, this reads better

File: docs/topics/extraction.md [hunk 3]
```diff
@@ -282,32 +274,25 @@ def query(base_url, entity, config_option):
...
 
-Some potential starting points:
+Here are some options to address this:
 
-* Remove the dependency on the core module or call them from the modules calling the utils module directly
-* Migrate the dependency into the Salt extension repository and modify it locally as described [here](utils-dunder-into-saltext-utils)
-* Submit a PR with the above changes to Salt core to be able to get rid of this code duplication at some point
+* Remove the dependency on the core module or call it from the modules calling the utils directly
+* Migrate the dependency into your Salt extension repository and modify it locally as described [here](utils-dunder-into-saltext-utils)
+* Submit a PR to Salt core with the necessary changes to elimiate the code duplication in the long term.
```
Comment (max-arnold) [inline docs/topics/extraction.md]: s/elimiate/eliminate/

**Comment by max-arnold** [comment]
I'm curious - what LLM and prompt did you use? Did it preserve the MyST syntax on its own, or you had to edit the output manually?

**Comment by lkubb** [comment]
> I'm curious - what LLM and prompt did you use?

Nothing fancy, just ChatGPT 4o with the following (lazy) prompt:

> I'm writing documentation around Salt extensions, which are pip-installable Python packages that can provide different modules to Salt (as in SaltStack, the automation platform).
>
> I'm going to send you one doc page at a time. Please optimize each page for clarity and conciseness.


> Did it preserve the MyST syntax on its own, or you had to edit the output manually?

It kept most of it, but sometimes dropped the roles like `{path}`/`{question}`. I still edited it manually side-by-side.

I had to break down some longer documents:

> Let's break the last document I sent down into parts. The previous modification is too concise at the cost of clarity.

Sometimes, I had to focus on some specific sentences. It initially dropped, then really tried to smoothen the following:

> After the Broadcom event, the push really came to shove, and it was communicated that there won't be any deprecation period

> Following the Broadcom event, it was announced that there won't be any deprecation period
> (me) The change is quite fundamental and abrupt, which your suggestion does not imply. Please rephrase
> After the Broadcom event, it was urgently announced that there won't be any deprecation period
> (me) please rephrase again, still too soft
> The Broadcom event forced an immediate shift in strategy, leading to the announcement that there won't be any deprecation period

I was quite impressed with its ability to interpret and follow spartan instructions tbh. For some lighter parts, its technical/rational tone did not match my intentions, so I had to re-prompt:

> (me) Please make this less clunky, but keep the vibe:
> (me) Very important: You don't need to publish inside the organization to use it!
It can be used for individual projects as well.
> Important: You don’t have to publish within the organization to use it — it's available for individual projects too!

For the home page:
> [turns the enumeration into a list]
> [...] In short, everything you need to manage your Salt extensions is right here!
> (me) Don't include a list, the navigation is on the same page
> [returns improved prose]
> (me) Please rephrase the last sentence while keeping the original vibe
> Wow, that’s quite a bit. But don’t worry—we’ve got you covered!
> (me) Again please. It's fine if you change the sentence heavily
> That’s a lot to take in, but don’t worry—we’ve got you covered every step of the way!

**Pull request merged**